### PR TITLE
#306 added file name to downloaded file

### PR DESCRIPTION
**Pull request by dmzaytsev**
For #306 
- added file name to downloaded file. According [RFC 2616 19.5.1](http://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html)
- modified the unit test

### CODE REVIEW

**Comment by k0r** [comment]
@yegor256 good to merge?

**Comment by karato** [comment]
@dmzaytsev Let me find a reviewer for this pull request, thanks for submitting it

**Comment by karato** [comment]
@longtimeago review it

File: src/test/java/com/nerodesk/takes/doc/TkReadTest.java [hunk 1]
```diff
@@ -59,16 +60,20 @@ public void readsFileContent() throws Exception {
...
-                    )
-                )
-            ).printBody(),
+            new RsPrint(resp).printBody(),
             Matchers.endsWith("world!")
         );
+        MatcherAssert.assertThat(
+            new RsPrint(resp).printHead(),
+            //@checkstyle LineLengthCheck (1 lines)
+            Matchers.containsString("Content-Disposition: attachment; filename=\"hey\"")
```
Comment (longtimeago) [inline src/test/java/com/nerodesk/takes/doc/TkReadTest.java]: @dmzaytsev would we need checkstyle suppression if put the string to the new line?

```
Matchers.containsString(
    "Content-Disposition: attachment; filename=\"hey\""
)
```

?

Comment (dmzaytsev) [inline src/test/java/com/nerodesk/takes/doc/TkReadTest.java]: @longtimeago sure, my bad,
 I'll fix it

**Comment by dmzaytsev** [comment]
@longtimeago I removed the checkstyle suppression

**Comment by longtimeago** [comment]
@dmzaytsev thank you!

**Comment by longtimeago** [comment]
@rultor merge

**Comment by k0r** [comment]
@yegor256 confirm?

File: src/main/java/com/nerodesk/takes/doc/TkRead.java [hunk 2]
```diff
@@ -64,7 +66,15 @@
     public Response act(final Request req) throws IOException {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
         new RqDoc(req, this.base).doc().read(baos);
-        return new RsWithBody(new ByteArrayInputStream(baos.toByteArray()));
+        return
+            new RsWithHeader(
```
Comment (yegor256) [inline src/main/java/com/nerodesk/takes/doc/TkRead.java]: @dmzaytsev what happened with indentation here? just put this `new RsWithHeader` to the previous line and remove checkstyle suppression

**Comment by yegor256** [comment]
@dmzaytsev looks good, but there is something wrong with indentation, see above

**Comment by dmzaytsev** [comment]
@yegor256 fixed, take another look please

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by karato** [comment]
@longtimeago Thanks so much! Your account was topped up for **17 mins** (transaction ID is `58609674`, the task took 8 hours and 7 mins). review comments (c=2) added as a bonus. +17 added to your rating, at the moment it is: [+3823](http://www.netbout.com/b/35024?open=rating)

**Comment by karato** [comment]
@rultor deploy pls

**Pull request merged**